scipy/spatial/transform/rotation.py

```diff
@@ -281,3 +282,147 @@ def as_rotvec(self):
             return rotvec[0]
         else:
             return rotvec
+
+    @classmethod
+    def _make_elementary_quat(cls, char, angles):
+        if len(char) != 1:
```

Comment:
My suggestion is to remove such checks from utility functions. It means that you should prepare/check everything in user API functions or methods before calling it.